Form/Type/SelectAjaxType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the She Interlang package
```

Comment:
please use the header defined in https://github.com/desarrolla2/FormBundle/blob/master/.formatter.yml